docs/connect.html

```diff
@@ -313,7 +313,13 @@ <h4><a id="connect_rest" href="#connect_rest">REST API</a></h4>
         <li><code>DELETE /connectors/{name}</code> - delete a connector, halting all tasks and deleting its configuration</li>
         <li><code>GET /connectors/{name}/topics</code> - get the set of topics that a specific connector is using since the connector was created or since a request to reset its set of active topics was issued</li>
         <li><code>PUT /connectors/{name}/topics/reset</code> - send a request to empty the set of active topics of a connector</li>
-        <li><code>GET /connectors/{name}/offsets</code> - get the current offsets for a connector (see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-875%3A+First-class+offsets+support+in+Kafka+Connect">KIP-875</a> for more details)</li>
+        <li>Offsets management REST APIs (see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-875%3A+First-class+offsets+support+in+Kafka+Connect">KIP-875</a> for more details):
+            <ul>
+                <li><code>GET /connectors/{name}/offsets</code> - get the current offsets for a connector</li>
+                <li><code>DELETE /connectors/{name}/offsets</code> - reset the offsets for a connector. The connector must exist and must be in the stopped state.</li>
```

Comment:
Nit: for consistency, we should leave out the period from the last sentence of each item
```suggestion
                <li><code>DELETE /connectors/{name}/offsets</code> - reset the offsets for a connector. The connector must exist and must be in the stopped state</li>
```

Also, would it be possible to link to the docs for the `PUT /connectors/{name}/stop` endpoint when we refer to it here?

Reply:
Hm the `PUT /connectors/{name}/stop` endpoint docs are in the same `connect_rest` section as these docs and I don't think it is possible to link to individual list items inside the section?

Reply:
It's possible; I'll add some GH suggestions demonstrating how.